tools/fuzz_test.go

```diff
@@ -0,0 +1,38 @@
+// Copyright 2020 The Gitea Authors. All rights reserved.
```

Comment:
```suggestion
// Copyright 2023 The Gitea Authors. All rights reserved.
```

Reply:
IMO, the year shouldn't be changed. It's just a file renaming.

Reply:
Code was changed...